addons/calendar/static/src/xml/base_calendar.xml

```diff
@@ -41,6 +41,9 @@
                 </div>
             </div>
         </t>
+        <t t-jquery=".o_cw_popover_fields_secondary, .o_cw_popover_edit, .o_cw_popover_delete" t-operation="attributes">
+            <attribute name="t-if">widget.eventPrivacy</attribute>
+        </t>
```

Comment:
`t-operation="attributes" replaces the old attribute value`. This is especially a problem with `.o_cw_popover_delete` that already has a `t-if="widget.canDelete"`.